CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+## Development
+
+For a development install (requires npm version 4 or later), do the following in the repository directory:
+
+```bash
+python -m pip install -e .
+jupyter serverextension enable jupyterlab_markup
+jlpm bootstrap
+jupyter labextension install .
+```
+
+To rebuild the package and the JupyterLab app:
+
+```bash
+jlpm build
+jupyter lab build
+```
+
+For live development
+
+```bash
+jlpm watch           # will continue running
+jupyter lab watch    # in another terminal, will continue running
+```
+
+## Extending
+
+Additional [`markdown-it` plugins][plugins]
+can be added as small labextensions. After getting started with the [official cookiecutter][],
```

Comment:
this will be much easier once we can `pip install` labextensions!

Reply:
for _users_, yes. _contributors_ will need _even more_ webpack stuff, and will have to track to a specific version of a jupyterlab-specific package. however, the dev docs for this repo will still be pretty simple, with the addition of `jupyter labextension build .`.